core/services/relay/evm/chain_reader.go

```diff
@@ -190,6 +195,24 @@ func (cr *chainReader) addEncoderDef(contractName, methodName string, method abi
 	return nil
 }
 
+func (cr *chainReader) addEventEncoderDef(contractName, methodName string, event abi.Event, chainReaderDefinition types.ChainReaderDefinition) error {
+	// prepend event sig as encodingPrefix
```

Comment:
Events don't go to chain, why would we ever have _input_ to `encode`?  We `decode` the _output_ given back to us.  

This should never be read either [GetLatestValue](https://github.com/smartcontractkit/chainlink/blob/BCF-2612-ChainReader-Next/core/services/relay/evm/binding.go#L120) for the event binding doesn't use the `codec` and the `ChainReader` does `decode` on the type with with `forEncoding` set to `false`